proto-google-cloud-bigtable-v2/clirr-ignored-differences.xml

```diff
@@ -16,4 +16,50 @@
     <className>com/google/bigtable/v2/*OrBuilder</className>
     <method>boolean has*(*)</method>
   </difference>
+  <difference>
```

Comment:
Do you mind adding a comment saying that this proto was renamed and was never exposed?

Reply:
done, also added a todo to remove this after the next release. Thanks!